recipes/trompeloeil/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+find_package(trompeloeil REQUIRED)
+
+add_executable(test_package example.cpp)
+target_link_libraries(test_package trompeloeil::trompeloeil)
+set_property(TARGET test_package PROPERTY CXX_STANDARD 14)
```

Comment:
Is this how I should do it, or can this be managed cleaner with conan?

Reply:
This is fine.

Reply:
Hi, @blackliner 

This is one of the TODOs that are pending in the infrastructure behind the scenes of this repository. Right now we are building many configurations, but all of them with the default for the C++ standard. It means that any recipe requiring any _modern_ C++ will be rejected because all the profiles will fail... the consequence is that people need to set explicitly the C++ standard in `CMakeLists.txt`. This is quite unfortunate because Conan knows nothing about it and it cannot provide valuable feedback to the users.

Adding these new configurations is something we will add after the major refactor we are currently doing to the infra, libraries and processes we are running. And I really expect these new profiles to increase the quality and usability of Conan recipes in this repository.

If you are interested in this topic, I wrote an example here (https://github.com/conan-io/examples/pull/73) with some recipes and how they could look like if we add those new configurations. And I think it provides a cleaner way to manage it. 

Meanwhile, this recipe is more than ok. Thanks!